chainer/functions/basic_math.py

```diff
@@ -12,6 +12,16 @@
 # Arithmetic
 # ------------------------------------------------------------------------------
 
+def _vectorize(x, typ):
```

Comment:
Should we move it to some public place as a utility? Other functions may want to use it.